contract-creator/src/org/jetbrains/ruby/runtime/signature/server/SignatureServer.kt

```diff
@@ -125,11 +147,22 @@ object SignatureServer {
             try {
                 val br = BufferedReader(InputStreamReader(socket.getInputStream()))
 
+                var iter = 0L
+                var millis = System.currentTimeMillis()
```

Comment:
These fields and corresponding logic are just logging/debugging goodness. I think it should be encapsulated into some logger class or something; in other case it's difficult to understand where business logic starts and debugging info does.